pkg/helmreconciler/reconciler.go

```diff
@@ -101,6 +101,12 @@ func (h *HelmReconciler) Reconcile() error {
 		return err
 	}
 
+	// filter empty manifests, which cannot be created or updated.
+	manifestMap, err = filterEmptyManifests(manifestMap)
```

Comment:
Seems like it would be easier to just return from the function that takes a manifest, if that manifest is empty.